### ISSUE TITLE
add setting to change font size

### ISSUE THREAD

**Issue by moates** [moates/onlyBans#20]
Player should be able to toggle font sizes - minimum needs to be 14pt

**Comment by moates** [comment]
https://github.com/moates/onlyBans/commit/f59b532e8172f6a051462a6fe369f0e373eee438

Using a range was okay, but as you click the range, the size of the screen content changes and it got real messy. The ideal option would be +/- buttons, but that isn’t built-in. Instead, I’m going to do a drop-down with a few options.

**Comment by moates** [comment]
apparently i didn't merge this yet

**Comment by moates** [comment]
still has a known bug #67  but i don't got time to figure that one out so gonna push this as good enough
